src/site/content/en/blog/decrease-frontend-size/index.md

```diff
@@ -0,0 +1,778 @@
+
+
+1. You write code like this:
+
+    ```js
+    // comments.js
+    import './comments.css';
+    export function render(data, target) {
+    console.log('Rendered!');
+    }
+    ```
+
+
+1. Webpack compiles it into approximately the following:
+
+    ```js
+    // bundle.js (part of)
+    "use strict";
+    Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
+    /* harmony export (immutable) */ __webpack_exports__["render"] = render;
+    /* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__comments_css__ = __webpack_require__(1);
+    /* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__comments_css_js___default =
+    __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_0__comments_css__);
+
+    function render(data, target) {
+    console.log('Rendered!');
+    }
+    ```
+
+1. A minifier compresses it into approximately the following:
+
+    ```js
+    // minified bundle.js (part of)
+    "use strict";function t(e,n){console.log("Rendered!")}
+    Object.defineProperty(n,"__esModule",{value:!0}),n.render=t;var o=r(1);r.n(o)
+    ```
+
+**In webpack 4,** the bundle-level minification is enabled automatically – both in the production
+mode and without one. It uses [the UglifyJS minifier](https://github.com/mishoo/UglifyJS2)
+under the hood. (If you ever need to disable minification, just use the development mode
+or pass `false` to the `optimization.minimize` option.)
+
+**In webpack 3,** you need to use [the UglifyJS plugin](https://github.com/webpack-contrib/uglifyjs-webpack-plugin)
+directly. The plugin comes bundled with webpack; to enable it, add it to the `plugins`
+section of the config:
+
+```js
+// webpack.config.js
+const webpack = require('webpack');
+
+module.exports = {
+    plugins: [
+    new webpack.optimize.UglifyJsPlugin(),
+    ],
+};
+```
+
+{% Aside %}
+In webpack 3, the UglifyJS plugin can’t compile the ES2015+ (ES6+) code. This means
+that if your code uses classes, arrow functions or other new language features,
+and you don’t compile them into ES5, the plugin will throw an error. <br><br>
+If you need to compile the new syntax, use the
+[uglifyjs-webpack-plugin](https://github.com/webpack-contrib/uglifyjs-webpack-plugin) package. This
+is the same plugin that’s bundled with webpack, but newer, and it’s able to compile the ES2015+
+code.
+{% endAside %}
+
+### Loader-specific options
+
+The second way to minify the code is loader-specific options ([what a loader
+is](https://webpack.js.org/concepts/loaders/)). With loader options, you can compress things that
+the minifier can’t minify. For example, when you import a CSS file with
+[`css-loader`](https://github.com/webpack-contrib/css-loader), the file is compiled into a string:
+
+
+```css
+/* comments.css */
+.comment {
+    color: black;
+}
+
+```
+
+```js
+/ minified bundle.js (part of)
+exports=module.exports=__webpack_require__(1)(),
+exports.push([module.i,<strong>".comment {\r\n  color: black;\r\n}"</strong>,""]);
```

Comment:
```suggestion
exports.push([module.i,".comment {\r\n  color: black;\r\n}",""]);
```

Reply:
Not sure if there was intended emphasis here, but it won't have the intended effect in a code block. It'll just render literally as `<strong>`.